### PR TITLE
TEAMNADO-5653 "Remove App_Navigation Event"

### PR DESCRIPTION
**Pull request by BrooklynLCodes**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -21,18 +21,12 @@ const queryClient = new QueryClient({
...
-    const unregister = insights.chrome.on('APP_NAVIGATION', (event) => {
-      const partialURL = getPartialRouteFromPath(event.domEvent.href);
-      navigate(partialURL);
-    });
-    return () => {
-      unregister();
-    };
+  useEffect(() => {
+    const registry = getRegistry();
+    registry.register({ notifications: notificationsReducer });
```
[the remark this answers is not in the document]
Comment (BrooklynLCodes) [reply src/App.js:28]: From my understanding in the examples provided for removing this event the getRegistry() function retrieves a registry and registers that 'notificationsReducer' to handle the notifications.

[the remark this answers is not in the document]
Comment (BrooklynLCodes) [reply src/App.js:28]: I have been using this PR which was tagged in Router v6 and nav event removal email for reference :https://github.com/gkarat/ocp-advisor-frontend/pull/101/files

[the remark this answers is not in the document]
Comment (jlgre) [reply src/App.js:28]: Yeah, I guess my question was more:
- What is a notificationReducer? In this instance, what is a registry?
- Do we need that? We handle notifications a different way

It looks like that PR already had that code and it wasn't a part of the removal of this event, maybe we need it too but if so, what does it actually do?

[the remark this answers is not in the document]
Comment (BrooklynLCodes) [reply src/App.js]: I understand your POV, I saw best to remove this function as it is not a. necessity in our code base

File: package.json [hunk 2]
```diff
@@ -83,6 +83,7 @@
     "@patternfly/react-core": "^4.276.8",
     "@patternfly/react-table": "^4.113.0",
     "@redhat-cloud-services/frontend-components": "^3.9.29",
+    "@redhat-cloud-services/frontend-components-notifications": "^3.2.14",
```
Comment (jlgre) [inline package.json:86]: We probably can uninstall this package now right?

**Review by jlgre — APPROVED** [review]

**Pull request merged**